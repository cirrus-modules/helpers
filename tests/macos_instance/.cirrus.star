load("../../lib.star", "task", "macos_instance")

def main(ctx):
    return [
        task("macos task", macos_instance("big-sur-xcode")),
    ]
