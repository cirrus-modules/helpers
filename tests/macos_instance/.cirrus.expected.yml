task:
  name: macos task
  macos_instance:
    image: big-sur-xcode
